sources/Algo/src/Algo.cpp

```diff
@@ -81,16 +83,53 @@ GeneratorDefinitionAlgorithm::operator()(const NodePtr& node) {
   if (node_generators.size() == 1) {
     auto model = useInfiniteReactivelimits_ ? GeneratorDefinition::ModelType::SIGNALN : GeneratorDefinition::ModelType::DIAGRAM_PQ_SIGNALN;
     const auto& gen = node_generators.front();
+
+    if (model == GeneratorDefinition::ModelType::DIAGRAM_PQ_SIGNALN && !isDiagramValid(gen)) {
+      return;
```

Comment:
Please add a warning saying that the default model willl be used for this generator

Reply:
In log(debug) or log(warning) ?

Reply:
We already have a warning when the diagram is invalid, so should I extend the original warning instead ?

Reply:
the consequence of this invalidity should also be given